pom.xml

```diff
@@ -68,7 +68,7 @@
 
         <!-- Test dependencies -->
         <strimzi.version>0.45.0</strimzi.version>  <!-- when bumping strimzi, be sure to comment out the kafka.version override in the systemtest module's pom -->
-        <kubernetes-client.version>6.13.5</kubernetes-client.version>
+        <kubernetes-client.version>7.1.0</kubernetes-client.version>
```

Comment:
curious, is the upgrade required by this work.  I'm also curious why dependabot hasn't done it already.

Reply:
It is not required. I had it locally because I realized dependabot didn't update it. Maybe it is worth it to have in a different PR.